src/components/imageContainer/Index.jsx

```diff
@@ -14,7 +16,7 @@ const ImageContainer = (props) => {
     <Container className="image-container">
       <Row>
         <Col style={{ textAlign: 'center' }}>
-          <Image fluid="true" src={src} style={{ width: '80%', height: '100%' }} />
+          <Image fluid="true" src={src} className="img" />
```

Comment:
Are we sure this `fluid` here should be a string or an actual boolean?

Reply:
I don't know. My changes on that line involve only the CSS styling.

Reply:
I have made the correction. By leaving it as just `fluid` it defaults to true.